safe/definitions/post_processors/__init__.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+from population_post_processors import *
```

Comment:
__[flake8]__
* [F403] 'from population_post_processors import *' used; unable to detect undefined names

<!-- Comment by [SideCI](https://sideci.com) -->